File: backend/src/Application/Features/Brokers/GetById/BrokerDetailsDto.cs
using RealEstate.Domain.Enums.Brokers;
    
namespace RealEstate.Application.Features.Brokers.GetById;

public sealed record BrokerDetailsDto(
    int Id,
    int AgencyId,
    string FirstName,
    string LastName,
    string Email,
    string PhoneNumber,
    string? PhotoUrl,
    BrokerGender Gender,
    DateTime CreatedAt,
    DateTime UpdatedAt
);

File: frontend/src/features/brokers/api/dtos/broker-details.dto.ts
export type BrokerGender = "Unspecified" | "male" | "female" | "other";

export type BrokerDetailsDto = {
  id: number
  agencyId: number
  firstName: string
  lastName: string
  email?: string
  phoneNumber?: string
  photoUrl?: string
  createdAt: string
  updatedAt: string
  gender?: BrokerGender
}
